sopel/modules/reddit.py

```diff
@@ -8,19 +8,21 @@
 
 https://sopel.chat
 """
-from __future__ import unicode_literals, absolute_import, print_function, division
+from __future__ import absolute_import, division, print_function, unicode_literals
 
 import datetime as dt
 import re
 import sys
 import textwrap
 
 import praw
+
 import prawcore
+
 import requests
```

Comment:
The extra blank lines annoy me. Maybe this would be sorted (ha) by switching to "google" order style, like I did in my WIP branch?